src/serviceLauncher.ts

```diff
@@ -40,7 +40,6 @@ function main() {
                 `node node_modules/@ceramicnetwork/cli/bin/ceramic daemon \
                     --verbose \
                     --log-to-files \
-                    --port ${service.port} \
```

Comment:
can we just comment it out for now and add a reference to https://github.com/ceramicnetwork/js-ceramic/issues/1681?

Reply:
Have to remove it because it is part of a multiline exec command but I copied and commented the full original command below.